src/components/formio/checkbox.tsx

```diff
@@ -4,21 +4,31 @@ import {Field, FormikHandlers, useFormikContext} from 'formik';
 import {useValidationErrors} from '@/utils/errors';
 
 import Component from './component';
+import Description from './description';
 import Tooltip from './tooltip';
 
 export interface CheckboxProps {
   name: string;
   label?: React.ReactNode;
+  required?: boolean;
   tooltip?: string;
+  description?: string;
```

Comment:
may also be a `React.ReactNode`, right?

Reply:
It is used this way: `{description && <Description text={description} />}` (and in other components as well)

Reply:
yeah, with:

```tsx
interface DescriptionProps {
  text: React.ReactNode;
}
```
but I see it's indeed inconsistent in existing components, so let's leave it a narrow type and widen it if needed.